test/kokoro/xds.sh

```diff
@@ -13,7 +13,13 @@ shopt -s extglob
 branch="${branch//[[:space:]]}"
 branch="${branch##remotes/origin/}"
 shopt -u extglob
-go build
+# Install a version of Go supported by gRPC for the new features, e.g.
+# errors.Is()
+curl --retry 3 -O -L https://go.dev/dl/go1.17.3.linux-amd64.tar.gz
+sudo tar -C /usr/local -xvf go1.17.3.linux-amd64.tar.gz
+sudo cp /usr/local/go/bin/go /usr/bin/go
```

Comment:
`sudo ln -s /usr/local/go/bin/go /usr/bin/go`? Seems like it should work, but not 100% sure.

Reply:
done